### PR TITLE
[doc] Point to Ibex documentation for verification status

### PR DESCRIPTION
**Pull request by GregAC**
Signed-off-by: Greg Chadwick <gac@lowrisc.org>

### CODE REVIEW

File: hw/ip/rv_core_ibex/data/rv_core_ibex.prj.hjson [hunk 1]
```diff
@@ -10,6 +10,7 @@
     version:            "1.0",
     life_stage:         "L1",
     design_stage:       "D2S",
-    verification_stage: "V1",
+    verification_stage: "N/A",
```
Comment (mundaym) [inline hw/ip/rv_core_ibex/data/rv_core_ibex.prj.hjson:13]: I think the status would be clearer if we marked this as V2S and then still have checklist headers for each of the verification stages.

For example:

```
### V1

The V1 checklist may be found in the [Ibex documentation](https://ibexcore.readthedocs.io/en/latest/03_reference/verification_stages.html#v1-checklist).

### V2

The V2 checklist may be found in the [Ibex documentation](https://ibexcore.readthedocs.io/en/latest/03_reference/verification_stages.html#v2-checklist).

### V2S

The V2S checklist may be found in the [Ibex documentation](https://ibexcore.readthedocs.io/en/latest/03_reference/verification_stages.html#v2s-checklist).
```

File: hw/ip/rv_core_ibex/doc/checklist.md [hunk 2]
```diff
@@ -127,148 +127,24 @@ Review        | [REVIEW_SW_ERRATA][]    | Not Started |
...
-[TB_LINT_SETUP]:                      {{<relref "/doc/project/checklist.md#tb_lint_setup" >}}
-[PRE_VERIFIED_SUB_MODULES_V1]:        {{<relref "/doc/project/checklist.md#pre_verified_sub_modules_v1" >}}
-[DESIGN_SPEC_REVIEWED]:               {{<relref "/doc/project/checklist.md#design_spec_reviewed" >}}
-[TESTPLAN_REVIEWED]:                  {{<relref "/doc/project/checklist.md#testplan_reviewed" >}}
-[STD_TEST_CATEGORIES_PLANNED]:        {{<relref "/doc/project/checklist.md#std_test_categories_planned" >}}
-[V2_CHECKLIST_SCOPED]:                {{<relref "/doc/project/checklist.md#v2_checklist_scoped" >}}
+### V1
 
-### V2
+The V1 checklist my be found in the [Ibex documentation](https://ibex-core.readthedocs.io/en/latest/03_reference/verification_stages.html#v1-checklist)
```
Comment (mundaym) [inline hw/ip/rv_core_ibex/doc/checklist.md:138]:
```suggestion
The V1 checklist may be found in the [Ibex documentation](https://ibex-core.readthedocs.io/en/latest/03_reference/verification_stages.html#v1-checklist).
```

**Review by mundaym — APPROVED** [review]
LGTM but please give @msfschaffner a chance to comment before merging.

**Review by msfschaffner — APPROVED** [review]
LGTM!

**Pull request merged**